.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.yarn
+.env
```

Comment:
Add a new line at the end of every file. Reason and how to fix it for once -> https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

Reply:
Resolved